plugins/ubiquiti/unifi_api

```diff
@@ -0,0 +1,1166 @@
+#!/usr/bin/perl
+# -*- perl -*-
+
+=encoding utf8
+
+=head1 NAME
+
+unifi_api - Munin plugin to display device and network information from the 
+            Ubiquiti unifi API
+
+=head1 APPLICABLE SYSTEMS
+
+Unifi controllors with direct API access
+
+Controller version 5+ required (tested with 5.8.x)
+
+WebRTC is not supported at this time
+
+=head1 CONFIGURATION
+
+This script uses the multigraph functionality to generate many graphs.
+
+You will need to supply your API login details:
+
+  [unifi_api]
+    env.user Controller_Username
+    # default is ubnt
+
+    env.pass Controller_Password
+    # default is ubnt
+
+    env.api_url https://unifi.fqdn.com:8443
+    # default is https://localhost:8443
+
+    env.ssl_verify_host no 
+    # Check That SSL host is valid, default is yes
+
+    env.ssl_verify_peer no 
+    # Check That SSL peer is valid, default is yes
+
+    env.name Site Name
+    # A pretty name for the unifi site - used in graph titles.
+
+    env.site site_string 
+    # default is "default" - found when you connect to the web interface
+    # it's the term in the URL - /manage/site/site_string/dashboard
+
+There are a number of possible options to turn off individual graphs.
+
+By default, everything is enabled.  Set to "no" to disable
+
+    env.enable_device_cpu yes
+    # Show device CPU utilization
+
+    env.enable_device_mem yes
+    # Show device memory usage
+
+    env.enable_device_load yes
+    # Show device load average (switches and APs only)
+
+    env.enable_device_uptime yes
+    # Show device uptime
+
+    env.enable_clients_device yes
+    # Show number of clients connected to each device
+    env.enable_detail_clients_device yes
+    # Show detailed graphs for each device
+
+    env.enable_clients_type yes
+    # Show number of clients connected to each network type
+    env.enable_detail_clients_type yes
+    # Show detailed graphs for each client type
+    env.show_authorized_clients_type yes
+    # Show unauthorized / authorized client list
+    # if you are not using the guest portal, this is useless
+
+    env.enable_xfer_port yes
+    # Show transfer statistics on switch ports
+    env.enable_detail_xfer_port yes
+    # Show detailed graphs per switch port
+    env.hide_empty_xfer_port yes
+    # Hide ports that have no link
+
+    env.enable_xfer_device yes
+    # Show transfer statistics per device
+    env.enable_detail_xfer_device yes
+    # Show detailed graphs for each device
+
+    env.enable_xfer_network yes
+    # Show transfer statistics per named network
+    env.enable_detail_xfer_network yes
+    # Show detailed graphs for each named network
+
+    env.enable_xfer_radio yes
+    # Show transfer statistics per radio
+    env.enable_detail_xfer_radio yes
+    # Show detailed graphs for each radio
+
+    env.use_well_know_categories yes
```

Comment:
This setting's name is probably not easy to understand for non-developers.
How about reverting the logic and using the setting name `force_category` (default: unset; overrides the `category` filed of all plugins if non-empty). In your case you would use `env.force_category unify`.

Reply:
yes, agreed.  In point of fact, it didn't actually work before - I wasn't reading the variable correctly.  Now called "force_category", as suggested.